src/main/java/frc/robot/utils/CRTGearRatioUtil.java

```diff
@@ -0,0 +1,154 @@
+package frc.robot.utils;
+
+import java.util.Optional;
+
+/* Helpful CRT functions - calculate the absolute position of an object attached to a gear
+ * 
+ * How to use
+ * -Initialize the crtutil using the init function
+ * -Then, to get the absolute location of the object attached to the gear, 
+ * call getAbsolutePositionOfMainObject on the util object with the values from the two gear
+ * 
+ */
+public class CRTGearRatioUtil {
+  private final double mainGearRotationRatioTerm;
+  private final double mainGearRotationRatioTermSimplified;
+  private final double secondaryGearRotationRatioTermSimplfied;
+  private final double secondaryGearRotationRatioTerm;
+  private final int maxRotations;
+  private final double mainGearCircumference;
+
+  /**
+   * 
+   * @param mainGearRotationRatioTerm      for every mainGearRotationRatioTerm
+   *                                       rotations of the main gear,
+   *                                       the secondary gear rotates
+   *                                       secondaryGearRotationRatioTerm times
+   * @param secondaryGearRotationRatioTerm for every mainGearRotationRatioTerm
+   *                                       rotations of the main gear,
+   *                                       the secondary gear rotates
+   *                                       secondaryGearRotationRatioTerm times
+   * @param maxRotations                   the maximum number of rotations that
+   *                                       the main gear will make that involve
+   *                                       unique positions of the object
+   *                                       attached to it
```

Comment:
Is this the desired max rotations, or the max rotations that the gear ratio allows? A little confused here. Double check how this is used because it seems like it's being used for both?

Reply:
fixed